docs/requirement-sets/powerpoint/powerpoint-api-1-5-requirement-set.md

```diff
@@ -0,0 +1,23 @@
+---
+title: PowerPoint JavaScript API requirement set 1.5
+description: Details about the PowerPointApi 1.5 requirement set.
+ms.date: 11/11/2022
+ms.prod: powerpoint
+ms.localizationpriority: medium
+---
+
+# What's new in PowerPoint JavaScript API 1.5
+
+PowerPointApi 1.5 added APIs to select slides, text ranges, and shapes within presentations. Prior to this release, selecting slides and shapes required a complicated series of steps. These selection APIs enable developers to directly select slides, text ranges, and shapes, creating a more efficient and intuitive add-in development process.
```

Comment:
@Rick-Kirkham Can you take a look at this article in particular? Want to make sure I covered the main benefits.